### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Morel-Victor**
Add home page and resolve conflict in back

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
OK for me :100:

File: routes/home.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+router.get('/', async (req, res) => {
+  try {
+    const [home] = await db.query(`
+    SELECT source, image, description, date
+    FROM news
+  `);
+    res.json(home);
+  } catch (err) {
+    res.status(404).send();
```
Comment (Hacene010) [inline routes/home.js]: Why did you keep the send() if it's empty ??

**Review by Hacene010 — CHANGES_REQUESTED** [review]
one small evil !

File: routes/home.js (hunk 1 above)
Comment (Morel-Victor) [reply routes/home.js]: i have remove .send

**Review by Caroline-CD — APPROVED** [review]
OK for :100:

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**